.github/workflows/build_base_image.yaml

```diff
@@ -0,0 +1,28 @@
+# Copyright (c) 2023 - 2023, Oracle and/or its affiliates. All rights reserved.
+# Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl/.
+
```

Comment:
Please add documentation on top to describe the purpose of this workflow.

Reply:
Fixed in https://github.com/oracle-samples/macaron/commit/d2fe70dc8bc12b52071d18e885bf63b86f9dcc25